config.ini

```diff
@@ -17,7 +17,7 @@ webpush_channels.permission_backend = kinto.core.permission.memory
 [server:main]
 use = egg:waitress#main
 host = 127.0.0.1
-port = 9999
+port = 8888
```

Comment:
I'll change it to 9999 again. Actually, the docs said 8888, so I changed this to 8888. I'll make the required changes in both the docs and webpush_channels